.github/workflows/behat.yaml

```diff
@@ -0,0 +1,167 @@
+
+            - name: Theme - Config Webpack Encore
+              working-directory: ./sylius
+              run: |
+                  echo "    builds:" >> config/packages/webpack_encore.yaml
+                  echo "        bootstrapTheme: '%kernel.project_dir%/public/bootstrap-theme'" >> config/packages/webpack_encore.yaml
+
+            - name: Theme - Build
+              working-directory: ./sylius
+              run: yarn build
+
+            - name: Theme - Set in Sylius Standard
+              working-directory: ./sylius
+              run: |
+                  sed -i "82i\                                theme_name: 'sylius/bootstrap-theme'" vendor/sylius/sylius/src/Sylius/Bundle/CoreBundle/Resources/config/app/fixtures/shop_configuration.yaml
+                  sed -i "113i\        \$channel->setThemeName('sylius/bootstrap-theme');" vendor/sylius/sylius/src/Sylius/Component/Core/Test/Services/DefaultUnitedStatesChannelFactory.php
+                  sed -i "88i\        \$channel->setThemeName('sylius/bootstrap-theme');" vendor/sylius/sylius/src/Sylius/Component/Core/Test/Services/DefaultChannelFactory.php
+
+            - name: Sylius - Init database
+              working-directory: ./sylius
+              run: php bin/console doctrine:database:create --if-not-exists
+
+            - name: Sylius - Install
+              working-directory: ./sylius
+              run: php bin/console sylius:install -n -s default
+
+            - name: Behat - Configure display
+              working-directory: ./sylius
+              run: |
+                  /sbin/start-stop-daemon --start --quiet --pidfile /tmp/xvfb_99.pid --make-pidfile --background --exec /usr/bin/Xvfb -- :99 -ac -screen 0 1680x1050x16
+                  export DISPLAY=:99
+
+            - name: Behat - Download and configure ChromeDriver
+              working-directory: ./sylius
+              run: |
+                  if [ ! -f chromedriver ] || [ "$(chromedriver --version | grep -c 2.34)" = "0" ]; then
+                      curl http://chromedriver.storage.googleapis.com/2.34/chromedriver_linux64.zip > chromedriver.zip
+                      unzip chromedriver.zip
+                      chmod +x chromedriver
+                  fi
+
+            - name: Behat - Run ChromeDriver
+              working-directory: ./sylius
+              run: chromedriver > /dev/null 2>&1 &
+
+            - name: Behat - Download and configure Selenium
+              working-directory: ./sylius
+              run: |
+                  if [ ! -f selenium.jar ] || [ "$(java -jar selenium.jar --version | grep -c 3.4.0)" = "0" ]; then
+                      curl http://selenium-release.storage.googleapis.com/3.4/selenium-server-standalone-3.4.0.jar > selenium.jar
+                  fi
+
+            - name: Behat - Run Selenium
+              working-directory: ./sylius
+              run: java -Dwebdriver.chrome.driver=chromedriver -jar selenium.jar > /dev/null 2>&1 &
```

Comment:
Could be nice to use the same tools as @pamil in this PR: https://github.com/Sylius/Sylius/pull/11509/files#diff-71de669a7b7453dc2677f55b6d69bdc9R10

Reply:
Job done, just need a test ;)